reference-client/src/test/java/com/transformuk/hee/tis/reference/client/impl/ReferenceServiceImplTest.java

```diff
@@ -1261,56 +1266,56 @@ public void shouldCheckCurrentTitleExistsWhenCurrentOnly() {
   }
 
   @Test
-  public void shouldCheckAnyRolesExistsWhenNotCurrentOnly() {
+  public void shouldCheckAnyRolesMatchesWhenNotCurrentOnly() {
```

Comment:
Would you mind restoring the old tests for exists() and implementing new tests for matches() - that way the code coverage won't decrease.

Reply:
I've restored them for both ReferenceServiceImpl and RoleResource. Originally I was going to simply change the old endpoint into the new one, and that's why I simply amended the old tests to fit the new endpoint.  Then I put the old endpoint back in because otherwise TCS would break until we merge its PR to make it use the new endpoint.
We may end up deleting these old tests though if we go back to REFERENCE later to delete the old endpoint once TCS passed to the new one.